Add Fastlane Plugin option to RN Hermes sourcemaps page (#7175)
getsentry/sentry-docs: src/platforms/react-native/manual-setup/hermes.mdx

When uploaded, the `index.android.bundle` file will be sized at 0 bytes. This is expected with Hermes bytecode and won't affect the source maps resolution.

</Note>

### Upload using Sentry Fastlane Plugin

To upload generated sourcemaps using Fashlane you can use `sentry_upload_sourcemap` action. See how to install Sentry Fastlane Plguin [here](https://github.com/getsentry/sentry-fastlane-plugin).

```ruby
sentry_upload_sourcemap(
  auth_token: 'YOUR_AUTH_TOKEN',
  org_slug: '___ORG_SLUG___',
  project_slug: '___PROJECT_SLUG___',
  version: '...',
  app_identifier: '...', # pass in the bundle_identifer of your app
  build: '...', # Optionally pass in the build number of your app
  dist: '...', # optional distribution of the release usually the buildnumber

>   sourcemap: ['main.jsbundle', 'main.jsbundle.map'], # Sourcemap(s) to upload. Path(s) can be a comma-separated string or an array of strings.

Reviewer: What's about the Android ones? shouldn't they be added as well?

When uploaded, the `index.android.bundle` file will be sized at 0 bytes. This is expected with Hermes bytecode and won't affect the source maps resolution.

</Note>

> ### Upload using Sentry Fastlane Plugin

Reviewer 2: ```suggestion
### Upload Using Sentry Fastlane Plugin
```

> To upload generated sourcemaps using Fashlane you can use `sentry_upload_sourcemap` action. See how to install Sentry Fastlane Plguin [here](https://github.com/getsentry/sentry-fastlane-plugin).

Reviewer 2: ```suggestion
To upload generated sourcemaps using Fastlane, use the `sentry_upload_sourcemap` action. See how to install Sentry Fastlane Plugin [here](https://github.com/getsentry/sentry-fastlane-plugin).
```
